mobile_verifier/src/heartbeats.rs

```diff
@@ -158,45 +212,47 @@ pub struct Heartbeat {
     pub cell_type: Option<CellType>,
     pub hotspot_key: PublicKeyBinary,
     pub timestamp: DateTime<Utc>,
+    pub coverage_object: Option<Uuid>,
+    pub lat: f64,
```

Comment:
Werent we avoiding the use of floats, especially when it came to the protos ?  That said i see these were previously defined as double in the proto..

Reply:
We might have been, but the proto has used floats for lat and lon in cellheartbeat message since the beginning